### PR TITLE
Add events

### PR DESCRIPTION
**Pull request by JAAR91**
- Use the logic provided by MIcroverse
- Implement the event handlers to use the math logic from the new files

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+}
+
+export default function calculate(obj, buttonName) {
+  if (buttonName === 'AC') {
+    return {
+      total: 0,
+      next: null,
+      operation: null,
+    };
+  }
```
Comment (DeliceLydia) [inline src/logic/calculate.js:14]: - [x] It is good to display a default zero on the screen when you open an application and after performing an operation when you press `AC` I recommend to display a zero in the screen as well

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    try {
+      return one.div(two).toString();
+    } catch (err) {
+      return "Can't divide by 0.";
+    }
+  }
+  if (operation === '%') {
+    return one.mod(two).toString();
+  }
+  throw Error(`Unknown operation '${operation}'`);
```
Comment (DeliceLydia) [inline src/logic/operate.js:25]: - [x] When I am performing an operation I am able to see the first number only let us say `8/2` I only see `8` and then I just see the answer I recommend that you display on the screen all the operation for a better user XP

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @JAAR91, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-  I recommend that you reduce the size of the calculator to fit on the screen currently I was not able to see everything on the screen it is big.


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by oxenprogrammer — APPROVED** [review]
Hi @JAAR91 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**